Add pager for comments on group overview

Martchus opened the pull request:
See https://progress.opensuse.org/issues/30400

![screenshot_20180123_123430](https://user-images.githubusercontent.com/10248953/35273687-c2c57354-0039-11e8-9d29-55de5baa9d59.png)

Still needs to be tested.

coolo commented on lib/OpenQA/WebAPI/Controller/Main.pm:
```diff
@@ -97,15 +97,37 @@ sub group_overview {
+            },
+            {
+                page     => $page,
+                rows     => 5,
```
the 5 is a big magic - and repeated here and in the template. Better pass it as param from the template and use a variable there.

Martchus replied:
True. I have to change that code anyways because having some "..." in the gap would be nice.

Martchus commented:
I fixed the magic 5 and added … between the gaps like the data tables do. The template code is still a bit messy, but it should work now. There's also a basic test.

Still we should figure out why the markdown rendering is so horribly slow.

okurz commented:
Awesome, thanks a lot
